src/doc/src/reference/unstable.md

```diff
@@ -1157,7 +1157,8 @@ You can use the instruction like this:
 
 ```rust,no_run
 // build.rs
-println!("cargo::rustc-check-cfg=cfg(foo, bar)");
```

Comment:
Btw, this is valid; this defines two different cfgs `foo` and `bar`, not one `foo="bar"`.

But I agree this isn't the best example.